caffe2/python/rnn_cell.py

```diff
@@ -1405,6 +1470,7 @@ def _LSTM(
 
 
 LSTM = functools.partial(_LSTM, LSTMCell)
```

Comment:
hm, this function is not doing just LSTM any more, maybe we should rename it

Reply:
to _RNN?

Reply:
done